src/HotChocolate/Core/test/Validation.Tests/MaxDepthRuleTestsSkipIntrospection.cs

```diff
@@ -0,0 +1,186 @@
+﻿using Microsoft.Extensions.DependencyInjection;
```

Comment:
I think there are some snapshots missing for this test class

Reply:
done